polling_stations/apps/addressbase/management/commands/update_addressbase.py

```diff
@@ -76,8 +76,8 @@ class Command(BaseCommand):
         "Updates both Addressbase and UPRN to Council mapping tables from local files"
     )
 
-    def __init__(self, stdout=None, stderr=None, no_color=False, force_color=False):
-        super().__init__(stdout, stderr, no_color, force_color)
+    def __init__(self):
+        super().__init__()
```

Comment:
I think I'd make this

```suggestion
    def __init__(self, *args, **kwargs):
        super().__init__(*args, **kwargs)
```